miden-lib/src/transaction/procedures.rs

```diff
@@ -0,0 +1,223 @@
+use alloc::vec::Vec;
+
+use miden_objects::{Digest, Felt, Hasher};
+
+use super::TransactionKernel;
+
+impl TransactionKernel {
+    /// Hashes of all dynamically executed kernel procedures.
+    pub const PROCEDURES: [Digest; 28] = [
```

Comment:
I wonder if we should define this as:
```rust
pub const PROCEDURES: [&[Digest]; 1] = [KERNEL0_PROCEDURES];
```
Where `KERNEL0_PROCEDURES` would basically be the same as the current `PROCEDURES`.

This way, when we add another version of the kernel, we could just add another entry to this array.